internal/cli/deployment_create.go

```diff
@@ -96,7 +100,7 @@ func (c *DeploymentCreateCommand) Run(args []string) int {
 		// Output
 		app.UI.Output("")
 		switch {
-		case releaseUrl != "":
+		case releaseUrl != "" || deployUrl != "":
```

Comment:
I think we need to add a `releaseUrl != ""` check in the output below so that we don't output an empty release URL.